event/event.go

```diff
@@ -2,12 +2,12 @@ package event
 
 import (
 	"github.com/ipld/go-ipld-prime"
-	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/libp2p/go-libp2p/core"
```

Comment:
Does this library provides all features of "github.com/libp2p/go-libp2p-core/peer" that we need here? @masih

Reply:
This library is a new version. Changes were made because there are many outdated objects. This change should not break functionality. It would be easier to test if there were unit tests.